lib/skate/detours/trip_modification.ex

```diff
@@ -0,0 +1,108 @@
+defmodule Skate.Detours.TripModification do
```

Comment:
I'm in the middle of writing a different comment about naming with relation to my ticket [⚙️🤝GTFS-TM:A module that takes modification events to send via MQTT](https://app.asana.com/0/1203014709808707/1207383192300471/f), but I realized while writing that that because this implements a GTFS spec, maybe this file should live at `Realtime.TripModification` since this is a GTFS-RT struct and not a Skate specific struct?

That also makes it worth thinking about if we should flatten all the "messages" into Modules within `Realtime`?

Reply:
fwiw, I still think having a [`new/1`](https://github.com/mbta/skate/pull/2665/files#r1648067277) function is worth it on this module, to make it easy to construct these, but I think it should live in `Realtime`

Reply:
[Done](https://github.com/mbta/skate/pull/2665/commits/28e0ef48eff87a249a4a6aa315cbba3abdc3ba39)

Reply:
I'm unresolving this because this thread included the `Realtime` recommendation as well, which I haven't done yet!

Stay tuned...

Reply:
[Realtime move!](https://github.com/mbta/skate/pull/2665/commits/df4871b05a174ee14241834ece7752874e93512d)

(Kind of fitting that I kept you updated about this change in real time, huh!)